src/Models/AZ3166Device.ts

```diff
@@ -206,8 +206,9 @@ export class AZ3166Device extends ArduinoDeviceBase {
       throw new Error('Unable to find the board in the config file.');
     }
 
-    // await ScaffoldGenerator.scaffolIoTProjectdFiles(this.projectFolder, this.vscodeFolderPath, 
-    //   this.boardFolderPath, this.devcontainerFolderPath, this.board.id);
+    const scaffoldGenerator = new ScaffoldGenerator();
+    await scaffoldGenerator.scaffolIoTProjectdFilesInWorkspace(this.projectFolder, this.vscodeFolderPath,

```

Comment:
scaffolIoTProjectdFilesInWorkspace  typo

what is the return value of this function?